src/views/TechnicalAssistance/TrbAdminTeamHome.tsx

```diff
@@ -0,0 +1,648 @@
+    row.original.requesterComponent
+  );
+}
+
+function TrbLeadCell({ row }: CellProps<TrbAdminTeamHomeRequest>) {
+  return getPersonVal(
+    row.original.trbLeadInfo.commonName,
+    row.original.trbLeadComponent
+  );
+}
+
+type TrbRequestsTableProps = {
+  requests: TrbAdminTeamHomeRequest[];
+};
+
+function TrbNewRequestsTable({ requests }: TrbRequestsTableProps) {
+  const { t } = useTranslation('technicalAssistance');
+
+  // @ts-ignore
+  const columns = useMemo<Column<TrbAdminTeamHomeRequest>[]>(() => {
+    return [
+      {
+        Header: t<string>('table.header.submissionDate'),
+        accessor: 'form.submittedAt',
+        Cell: SubmissionDateCell
+      },
+      {
+        Header: t<string>('table.header.requestName'),
+        accessor: 'name',
+        Cell: RequestNameCell
+      },
+      {
+        Header: t<string>('adminHome.requester'),
+        accessor: 'requesterInfo.commonName',
+        Cell: RequesterCell
+      },
+      {
+        Header: t<string>('adminHome.requestType'),
+        accessor: ({ type }) => t(`table.requestTypes.${type}`)
+      },
+      {
+        Header: t<string>('documents.table.header.actions'),
+        Cell: ({ value, row }: CellProps<TrbAdminTeamHomeRequest>) => {
+          return (
+            <>
+              <UswdsReactLink
+                to={`/trb/${row.original.id}/initial-request-form`}
+                className="margin-right-2"
+              >
+                {t('adminTeamHome.actions.reviewRequest')}
+              </UswdsReactLink>
+              {/* wip assign lead could be a modal popup */}
+              <Button type="button" unstyled>
+                {t('adminTeamHome.actions.assignLead')}
+              </Button>
+            </>
+          );
+        },
+        disableSortBy: true
+      }
+    ];
+  }, [t]);
+
+  const {
+    canNextPage,
+    canPreviousPage,
+    getTableBodyProps,
+    getTableProps,
+    gotoPage,
+    headerGroups,
+    nextPage,
+    page,
+    pageCount,
+    pageOptions,
+    prepareRow,
+    previousPage,
+    rows,
+    setPageSize,
+    state
+  } = useTable(
+    {
+      columns,
+      globalFilter: useMemo(() => globalFilterCellText, []),
+      data: requests,
+      autoResetSortBy: false,
+      autoResetPage: false,
+      initialState: {
+        sortBy: useMemo(() => [{ id: 'form.submittedAt', desc: true }], []),
+        pageIndex: 0,
+        pageSize: 5
+      }
+    },
+    useFilters,
+    useGlobalFilter,
+    useSortBy,
+    usePagination
+  );
+
+  rows.map(row => prepareRow(row));
+
+  return (
+    <div className="bg-accent-cool-lighter padding-4">
+      <h3 className="margin-top-0 margin-bottom-1">
+        {t('adminTeamHome.newRequests.heading')}
+      </h3>
+      <div className="margin-bottom-1 line-height-body-5">
+        {t('adminTeamHome.newRequests.description')}
+      </div>
+
+      {/* Download new requests csv */}
+      <div className="margin-bottom-4 line-height-body-5">
```

Comment:
Are the extra divs used around `<CsvDownloadLink>` and the tables just to add utility classes? It would cut down on extra divs to add a `className` prop to the components.

`<CsvDownloadLink>` would just need the `display-block` class added to get the margins to work.